config.js

```diff
@@ -71,11 +71,24 @@ const readReplicas = configUtil.getReadReplicas(pe, replicaConfigLabel);
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_ASYNC = 3600;
 const DEFAULT_JOB_QUEUE_TTL_SECONDS_SYNC = 25;
 
-// default set to 30 minutes
-const DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS = 1800;
+const DEFAULT_JOB_REMOVAL_INTERVAL_MINUTES = 30;
+const JOB_REMOVAL_INTERVAL = 60 * 1000 * (pe.KUE_JOBS_REMOVAL_INTERVAL_MINUTES ||
+  DEFAULT_JOB_REMOVAL_INTERVAL_MINUTES);
 
-const JOB_REMOVAL_INTERVAL_SECONDS = pe.KUE_JOBS_REMOVAL_INTERVAL ||
-  DEFAULT_JOB_REMOVAL_INTERVAL_SECONDS;
+const DEFAULT_JOB_REMOVAL_DELAY_MINUTES = 30;
+const JOB_REMOVAL_DELAY = 60 * 1000 * (pe.KUE_JOBS_REMOVAL_DELAY_MINUTES ||
+  DEFAULT_JOB_REMOVAL_DELAY_MINUTES);
+
+const DEFAULT_JOB_REMOVAL_WINDOW = 1000;
+const JOB_REMOVAL_WINDOW = pe.KUE_JOBS_REMOVAL_WINDOW ||
```

Comment:
those defaults seem fine to me

maybe change var name to use something like batch size instead of window?